app/views/spree/shared/_head.html.erb

```diff
@@ -0,0 +1,13 @@
+<meta charset="utf-8">
+<title><%= title %></title>
+<meta name="viewport" content="width=device-width, initial-scale=1">
+<%== meta_data_tags %>
+<%= canonical_tag(current_store.url) %>
+<%= csrf_meta_tags %>
+
+<%= favicon_link_tag 'favicon.ico' %>
+
+<%= stylesheet_link_tag 'spree/frontend/all', media: 'screen' %>
+<%= javascript_include_tag 'spree/frontend/all' %>
+
```

Comment:
here we removed retro compatibility
```
<!--[if lt IE 9]>
  <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6/html5shiv.min.js"></script>
<![endif]-->
```

https://github.com/solidusio/solidus/blob/master/frontend/app/views/spree/shared/_head.html.erb

Reply:
Retro compatibility is something we should not worry about at the moment. Maybe we should also discuss internally what level of support we want to have for this starter.